common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -263,7 +263,44 @@ public class CommonValue {
                         + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
                         + "      \""
                         + MLModel.GUARDRAILS_FIELD
-                        + "\" : {\"type\": \"flat_object\"},\n"
```

Comment:
Why not use flat_object? Isn't more flexible with flat_object?

Reply:
I got error like in description.